doc/manual/src/glossary.md

```diff
@@ -29,6 +51,17 @@
     store object by downloading a pre-built version of the store object
     from some server.
 
+  - [substituter]{#gloss-substituter}\
+    A *substituter* is a store other than `/nix/store` from which Nix will
+    copy a store path instead of building it.  Nix will not copy a store
+    path from a remote store unless one of the following is true:
+
+    - the store object is signed by one of the `trusted-public-keys`
+    - the substituter is in the `trusted-substituters` list
+    - the `no-require-sigs` option has been set to disable signature checking
+    - the store object is a derivation
+    - the store object is the realisation of a fixed-output derivation
```

Comment:
I don't think the glossary is the right place for this. A glossary should give brief definitions of terms, not a detailed description of the semantics of the store security model. Probably the manual needs a chapter on store path signing.

Reply:
Yeah it can be brief description + link to the section where the concept is describe in greater depth.

Reply:
> I don't think the glossary is the right place for this. A glossary should give brief definitions of terms, not a detailed description of the semantics of the store security model. Probably the manual needs a chapter on store path signing.

I agree.  Can you suggest someplace where I can move this list until the indefinite time at which that chapter materializes?

Reply:
I've moved it to the description of `--option substituter` for now.  If there's a better place just let me know.